### PR TITLE
Fix bug where user can login to game twice

### LINKED ISSUE #27: Need to handle when user is trying to be its own opponent

### PR DESCRIPTION
**Pull request by Drozd0f**

### CODE REVIEW

File: src/handlers.py [hunk 1]
```diff
@@ -71,6 +71,8 @@ async def login_game(request: web.Request) -> web.Response:
         return web.json_response({'message': 'game not found'}, status=404)
     if game.current_state != State.PENDING.value:
         return web.json_response({'message': 'invalid state'}, status=400)
+    if game.owner_id == request.user.id:
+        return web.json_response({'message': 'user already exist in game'}, status=404)
```
Comment (qerdcv) [inline src/handlers.py]: status 404?

Comment (qerdcv) [reply src/handlers.py]: And weird check.

If owner is in request => throw error that user is already logged in game, that's ok, but if another user tries to make few request to login, what will happen?

**Review by qerdcv — CHANGES_REQUESTED** [review]

File: src/handlers.py (hunk 1 above)
Comment (Drozd0f) [reply src/handlers.py]: status 400?

Comment (Drozd0f) [reply src/handlers.py]: "another user tries to make few request to login, what will happen" how i can emulate this?

Comment (qerdcv) [reply src/handlers.py]: Login as non-owner, and make requests login requests

Comment (qerdcv) [reply src/handlers.py]: Status conflict, user already in game

**Review by qerdcv — APPROVED** [review]

**Pull request merged**